xbmc/addons/InputStream.cpp

```diff
@@ -98,7 +98,7 @@ bool CInputStream::Supports(CFileItem &fileitem)
   CVariant addon = fileitem.GetProperty("inputstreamaddon");
```

Comment:
Is this documented or referenced elsewhere? I'm adding a property for game add-ons (probably "gameclient", open for debate) and I'm wondering where I should document it